MIGRATION.md

```diff
@@ -122,9 +122,9 @@ Installation should be global if newman is installed globally, local otherwise.
 
 The complete installation and usage guide is available here: https://github.com/postmanlabs/newman-reporter-html
 
-### Updating v1 Postman Collection format
-Newman >= v4 drops support for v1 collection format.<br/>
-Use [Postman native application](https://www.getpostman.com/apps) to convert v1 collection format to v2 collection format.
+### Updating the v1 Postman Collection format
```

Comment:
`Updating the v1 ... format` is misleading, let the title be `Dropped support for the v1 collection format`